src/Symfony/Component/AssetMapper/Tests/ImportMap/RemotePackageStorageTest.php

```diff
@@ -43,14 +43,14 @@ public function testGetStorageDir()
 
     public function testSaveThrowsWhenFailing()
     {
+        if ('\\' === \DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('This test cannot run on Windows.');
+        }
```

Comment:
Still got errors (on AppVeyor) with this test, the "chmod" function cannot be relied on
... and this is the only way to trigger the expected exception

So let's skip it on Windows (i based this on Filesystem tests doing the same thing)

Reply:
A new one (again the same problem i suppose of conflct between tests?)

Reply:
Update: let's try again after #58152